From 5a6604a7a2e52e5cd84c1f53724b3f7c325b5dff Mon Sep 17 00:00:00 2001
From: Pekka Paalanen <pekka.paalanen@collabora.com>
Upstream-Status: Backport
Date: Fri, 14 May 2021 15:54:56 +0300
Subject: [PATCH] libweston-desktop: rename weston_desktop_client_destroy()

This function here is a wl_resource destructor, but we will need another
function for externally triggered destroy when wl_resource does not
exist.

Rename the existing function, because the old name fits better the new
function to be written.

Signed-off-by: Pekka Paalanen <pekka.paalanen@collabora.com>
---
 libweston-desktop/client.c | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/libweston-desktop/client.c b/libweston-desktop/client.c
index 56413f713..ba7bfbc46 100644
--- a/libweston-desktop/client.c
+++ b/libweston-desktop/client.c
@@ -49,7 +49,7 @@ weston_desktop_client_add_destroy_listener(struct weston_desktop_client *client,
 }
 
 static void
-weston_desktop_client_destroy(struct wl_resource *resource)
+weston_desktop_client_handle_destroy(struct wl_resource *resource)
 {
 	struct weston_desktop_client *client =
 		wl_resource_get_user_data(resource);
@@ -117,13 +117,12 @@ weston_desktop_client_create(struct weston_desktop *desktop,
 
 	if (dispatcher != NULL)
 		wl_resource_set_dispatcher(client->resource, dispatcher,
-					   weston_desktop_client_destroy, client,
-					   weston_desktop_client_destroy);
+					   weston_desktop_client_handle_destroy, client,
+					   weston_desktop_client_handle_destroy);
 	else
 		wl_resource_set_implementation(client->resource, implementation,
 					       client,
-					       weston_desktop_client_destroy);
-
+					       weston_desktop_client_handle_destroy);
 
 	display = wl_client_get_display(client->client);
 	loop = wl_display_get_event_loop(display);
-- 
GitLab

